src/selectors/expandThoughts.js

```diff
@@ -30,7 +31,7 @@ import getThoughtsRanked from '../selectors/getThoughtsRanked'
     ...
   }
 */
-export const expandThoughts = (path, thoughtIndex, contextIndex, contextViews = {}, contextChain = [], { depth = 0 } = {}) => {
+export default ({ thoughtIndex, contextIndex, contextViews }, path, contextChain = [], { depth = 0 } = {}) => {
```

Comment:
I don't think we need set default, because of `initialState` contextViews is set empty object in initialState

Reply:
Not every selector is being called with the complete state. Though it might work in this case, I think unless the functions are type-safe, it's always good to have a fallback.